modules/validations/invalid_dashes.js

```diff
@@ -0,0 +1,150 @@
+import { t } from '../core/localizer';
+import { operationDelete } from '../operations';
+import { utilDisplayLabel } from '../util/utilDisplayLabel';
+import { validationIssue,validationIssueFix } from '../core/validation';
+import { actionChangeTags } from '../actions';
+export function validationDashes() {
+    var type = 'invalid_dashes';
+
+    var validation = function (entity) {
+        var issues = [];
+        function showReferenceDash(selection) {
+            selection.selectAll('.issue-reference')
+                .data([0])
+                .enter()
+                .append('div')
+                .attr('class', 'issue-reference')
+                .call(t.append('issues.invalid_dashes.reference'));
+        }
+    const DASH_SENSITIVE_KEYS = new Set([
```

Comment:
Lists like this go in modules/osm/tags.js. Once openstreetmap/id-tagging-schema#1537, we can remove this list in favor of checking for a field `type` of `schedule`.